src/Applications/LDAS_App/lenkf.j.template

```diff
@@ -38,7 +38,9 @@ endif
 unset argv
 setenv argv
 
-source $GEOSBIN/g5_modules
+if ( $debug_flag == 0 ) then
+  source $GEOSBIN/g5_modules
+endif
```

Comment:
This is meant to take care of @gmao-qliu's suggestion to avoid "source g5_modules" when in debug mode.  
If this works, then I'm not sure why we would need `unset argv; setenv argv` above that was introduced to avoid g5_modules tripping up when lenkf.j is called with the -debug flag.  Maybe the suggestion to omit "source g5_modules" here won't work?  I'm getting really confused.

Reply:
Since '-debug" option only prints out information, I would like to change it to "--help". It prints out and exits before source g5_modules

Reply:
Having said that, does the preprocessing possibly need "source g5_modules"? @gmao-qliu: Did you verify that we can skip "source g5_modules" when running with -debug?

Reply:
> 

Actually, the preprocessing is not necessary for debug. Except users change the directory to scratch and launch totalview

Reply:
> Having said that, does the preprocessing possibly need "source g5_modules"? @gmao-qliu: Did you verify that we can skip "source g5_modules" when running with -debug?

actually "source g5_modules" is needed with -debug because otherwise $BASEDIR is undefined.  I take my suggestion back.

Reply:
After users finish ldas_setup and get the interactive nodes, they can go to the "run" directory, source ../build/bin/g5_modules.sh , then load and launch totalview in run directory

Reply:
@weiyuan-jiang : Are you seriously saying that the first 200-300 lines in lenkf.j are not needed?  If that's the case, why can't we just delete them altogether?